### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by japborst**
To avoid updating the copywrite in the future, we can easily return the current year in Jekyll.

Whilst at it, I updated the docs wrt livereloading.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,33 @@
-Picnic Technologies Open Source Website
-=========================
+# Picnic Technologies Open Source Website
 
 Showcasing our latest open-source projects.
 
 Live at: [http://picnic.tech](http://picnic.tech)
```
Comment (werli) [inline README.md:5]: Always a good practice. :slightly_smiling_face: 
```suggestion
Live at: [https://picnic.tech](https://picnic.tech)
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
+   its name.
 
 ## Built using
-+ [GitHub Pages](https://pages.github.com/)
-+ [Jekyll](https://jekyllrb.com/)
-+ [Jekyll-grid theme](http://femmebot.github.io/jekyll-grid)
+
+- [GitHub Pages](https://pages.github.com/)
+- [Jekyll](https://jekyllrb.com/)
+- [Jekyll-grid theme](http://femmebot.github.io/jekyll-grid)
```
Comment (werli) [inline README.md:33]: Same thing. :+1: 
```suggestion
- [Jekyll-grid theme](https://femmebot.github.io/jekyll-grid)
```

File: _includes/footer.html [hunk 3]
```diff
@@ -4,8 +4,8 @@
     <footer class="footer">
       <div class="footer-description">
         <p><a href="https://blog.picnic.nl">[ Blog</a> | <a href="https://join.picnic.app">Join us ]</a></p>
-        <p>&copy; 2019 Picnic Technologies, All rights reserved.</p>
+        <p>&copy; {{ site.time | date: '%Y' }} Picnic Technologies, All rights reserved.</p>
```
Comment (werli) [inline _includes/footer.html:7]: Thanks @japborst :muscle:

**Review by werli — APPROVED** [review]
Thanks for the quick follow up @japborst :+1: 

Optimistically approving already. :rocket:

File: README.md (hunk 1 above)
Comment (japborst) [reply README.md:5]: Indeed 🔒

File: README.md (hunk 2 above)
Comment (japborst) [reply README.md:33]: We can, although that site doesn't have a certificate and Chrome tell me it's not secure.

Comment (werli) [reply README.md]: Works well on my Chrome
image

But I saw that you had already updated it. :+1:

**Pull request merged**